components/transaction/internal/adapters/postgres/operation/operation.postgresql.go

```diff
@@ -219,10 +236,21 @@ func (r *OperationPostgreSQLRepository) FindAll(ctx context.Context, organizatio
 	if err := rows.Err(); err != nil {
 		mopentelemetry.HandleSpanError(&span, "Failed to get rows", err)
 
-		return nil, err
+		return nil, http.CursorPagination{}, err
 	}
 
```

Comment:
In this follows lines are missing return arguments, please check.

L: 242
L: 785
L: 912

```suggestion
return nil, http.CursorPagination{}, err
```